src/playground/sample_data_tooling/Dockerfile

```diff
@@ -1,3 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
 # syntax=docker/dockerfile:1
```

Comment:
do you need this here?

Reply:
I pulled this from the header of your example Dockerfile. I'm not too sure if I need this

Reply:
I'm meaning that "# syntax=docker/dockerfile:1"

Reply:
Yes, I don't need it and I have removed it in the latest commit.